docs-en/chapter_introduction/algorithms_are_everywhere.md

```diff
@@ -4,11 +4,11 @@ When we hear the word "algorithm", we naturally think of mathematics. However, m
 
 Before we formally discuss algorithms, an interesting fact is worth sharing: **you have already learned many algorithms unconsciously and have become accustomed to applying them in your daily life**. Below, I will give a few specific examples to prove this point.
 
-**Example 1: Looking Up a Dictionary**. In a dictionary, each Chinese character corresponds to a pinyin, and the dictionary is arranged in alphabetical order of pinyin. Suppose we need to find a character whose pinyin starts with the letter $r$. This is usually achieved in the following way:
+**Example 1: Looking Up a Dictionary**. In a dictionary, each Chinese character corresponds to a pinyin, and the dictionary is arranged in alphabetical order of pinyin. Suppose we need to find a character with a pinyin begins with the letter $r$. This is typically accomplished in the following way:
```

Comment:
Just a suggestion, for English reader, can the examples be modified without using pinyin?